components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -380,6 +393,72 @@ void AdsServiceImpl::Start() {
   BackgroundHelper::GetInstance()->AddObserver(this);
 }
 
+void AdsServiceImpl::MaybeShowFirstLaunchNotification(bool rewards_enabled) {
```

Comment:
this also needs tests, both unit and browser tests

Reply:
Agreed to raise follow up issues with Ryan and will make sure these are prioritised